network/common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -56,6 +62,15 @@ public OneForOneStreamManager() {
     // This does not need to be globally unique, only unique to this class.
     nextStreamId = new AtomicLong((long) new Random().nextInt(Integer.MAX_VALUE) * 1000);
     streams = new ConcurrentHashMap<Long, StreamState>();
+    streamIds = new ConcurrentHashMap<Channel, Set<Long>>();
```

Comment:
Also, if you wouldn't mind, add a Preconditions.checkNotNull(buffers) to line 56 (to ensure buffers is not null).